server/plugin/api_test.go

```diff
@@ -1060,7 +1060,7 @@ func TestCreateSubscription(t *testing.T) {
 		ExpectedStatusCode   int
 		ExpectedErrorMessage string
 	}{
-		"success": {
+		"success with subscription type record": {
```

Comment:
we can do like this maybe ?
```suggestion
		"success with subscription type: record": {
```